src/components/footer-buttons/footer-buttons.html

```diff
@@ -23,6 +23,11 @@
           <ion-icon name="arrow-back"></ion-icon>{{'buttons.back'| translate}}
         </button>
       </ion-col>
+      <ion-col>
+        <!-- {{completedQuestionCount}} {{questionCount}} -->
+        <progress-bar [progress] = "percentage" [totalQuestionCount]="questionCount" [showQuestionTracker]="true" [completedQuestionCount]="completedQuestionCount" class="_full-width"></progress-bar>
```

Comment:
Make the input name  generic, like total, showTracker and completed so that we need not change the inputs when using in some other use cases